### ISSUE TITLE
when i use command pip install chatterot in my system.grtting this error

### ISSUE THREAD

**Issue by shivam2200** [gunthercox/ChatterBot#1702]
``` Bash
 Command "c:\python27\python.exe c:\python27\lib\site-packages\pip install --ignore-installed --no-user --prefix c:\users\acer\appdata\local\temp\pip-build-env-rwubk8\overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel>0.32.0.<0.33.0 Cython cymem>=2.0.2,<2.1.0 preshed>=2.0.1,<2.1.0 murmurhash>=0.28.0,<1.1.0 thinc==7.0.0.dev6" failed with error code 1 in None
```

**Comment by vkosuri** [comment]
The latest chatterbot package won't support ``python2.7`` please to install with python3

**Comment by shivam2200** [comment]
also grtting error in python 3.7 please help:(
  Command "python setup.py egg_info" failed with error code 1 in C:\Users\acer\AppData\Local\Temp\pip-install-67x4149h\thinc\

  ----------------------------------------
Command "c:\users\acer\appdata\local\programs\python\python37-32\python.exe c:\users\acer\appdata\local\programs\python\python37-32\lib\site-packages\pip install --ignore-installed --no-user --prefix C:\Users\acer\AppData\Local\Temp\pip-build-env-7wpj8xtg\overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel>0.32.0.<0.33.0 Cython cymem>=2.0.2,<2.1.0 preshed>=2.0.1,<2.1.0 murmurhash>=0.28.0,<1.1.0 thinc==7.0.0.dev6" failed with error code 1 in None

**Comment by Orfeous** [comment]
spaCy dropped 32bit support. If you try with 64 bit python it will work. See #1699

**Comment by pavankumar1809** [comment]
ERROR: Command errored out with exit status 1:
   command: 'c:\users\siri\appdata\local\programs\python\python38\python.exe' 'c:\users\siri\appdata\local\programs\python\python38\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0'
       cwd: None
  Complete output (105 lines):
  Collecting setuptools
    Using cached setuptools-46.1.3-py3-none-any.whl (582 kB)
  Collecting wheel<0.33.0,>0.32.0
    Using cached wheel-0.32.3-py2.py3-none-any.whl (21 kB)
  Collecting Cython
    Using cached Cython-0.29.16-cp38-cp38-win_amd64.whl (1.7 MB)
  Collecting cymem<2.1.0,>=2.0.2
    Using cached cymem-2.0.3-cp38-cp38-win_amd64.whl (33 kB)
  Collecting preshed<2.1.0,>=2.0.1
    Using cached preshed-2.0.1.tar.gz (113 kB)
  Collecting murmurhash<1.1.0,>=0.28.0
    Using cached murmurhash-1.0.2-cp38-cp38-win_amd64.whl (20 kB)
  Processing c:\users\siri\appdata\local\pip\cache\wheels\f2\32\f0\fe1467f30fde6dd38ef58010310d2f114b99db79391ff8c0cb\thinc-7.0.8-cp38-none-win_amd64.whl
  Collecting blis<0.3.0,>=0.2.1
    Using cached blis-0.2.4.tar.gz (1.5 MB)
  Collecting srsly<1.1.0,>=0.0.6
    Using cached srsly-1.0.2-cp38-cp38-win_amd64.whl (181 kB)
  Collecting plac<1.0.0,>=0.9.6
    Using cached plac-0.9.6-py2.py3-none-any.whl (20 kB)
  Collecting wasabi<1.1.0,>=0.0.9
    Using cached wasabi-0.6.0-py3-none-any.whl (20 kB)
  Collecting numpy>=1.7.0
    Using cached numpy-1.18.2-cp38-cp38-win_amd64.whl (12.8 MB)
  Collecting tqdm<5.0.0,>=4.10.0
    Using cached tqdm-4.44.1-py2.py3-none-any.whl (60 kB)
  Building wheels for collected packages: preshed, blis
    Building wheel for preshed (setup.py): started
    Building wheel for preshed (setup.py): finished with status 'done'
    Created wheel for preshed: filename=preshed-2.0.1-cp38-cp38m-win_amd64.whl size=79069 sha256=15a3ac51813eb22c7a2cb61b43baf482df743881465f94e632600b71bd460855
    Stored in directory: c:\users\siri\appdata\local\pip\cache\wheels\5a\d0\29\7f6993a759349eae3d0ecca7e2fbc88acdd8650b25e6c6ad8a
    Building wheel for blis (setup.py): started
    Building wheel for blis (setup.py): finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\siri\appdata\local\programs\python\python38\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"'; __file__='"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\Siri\AppData\Local\Temp\pip-wheel-bwvbk43j'
         cwd: C:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\
    Complete output (25 lines):
    BLIS_COMPILER? None
    running bdist_wheel
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.8
    creating build\lib.win-amd64-3.8\blis
    copying blis\about.py -> build\lib.win-amd64-3.8\blis
    copying blis\benchmark.py -> build\lib.win-amd64-3.8\blis
    copying blis\__init__.py -> build\lib.win-amd64-3.8\blis
    creating build\lib.win-amd64-3.8\blis\tests
    copying blis\tests\common.py -> build\lib.win-amd64-3.8\blis\tests
    copying blis\tests\test_dotv.py -> build\lib.win-amd64-3.8\blis\tests
    copying blis\tests\test_gemm.py -> build\lib.win-amd64-3.8\blis\tests
    copying blis\tests\__init__.py -> build\lib.win-amd64-3.8\blis\tests
    copying blis\cy.pyx -> build\lib.win-amd64-3.8\blis
    copying blis\py.pyx -> build\lib.win-amd64-3.8\blis
    copying blis\cy.pxd -> build\lib.win-amd64-3.8\blis
    copying blis\__init__.pxd -> build\lib.win-amd64-3.8\blis
    running build_ext
    error: [WinError 2] The system cannot find the file specified
    msvc
    py_compiler msvc
    {'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'HOSTTYPE': 'x86_64', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'LANG': 'C.UTF-8', 'OLDPWD': '/home/matt/repos/flame-blis', 'VIRTUAL_ENV': '/home/matt/repos/cython-blis/env3.6', 'USER': 'matt', 'PWD': '/home/matt/repos/cython-blis', 'HOME': '/home/matt', 'NAME': 'LAPTOP-OMKOB3VM', 'XDG_DATA_DIRS': '/usr/local/share:/usr/share:/var/lib/snapd/desktop', 'SHELL': '/bin/bash', 'TERM': 'xterm-256color', 'SHLVL': '1', 'LOGNAME': 'matt', 'PATH': '/home/matt/repos/cython-blis/env3.6/bin:/tmp/google-cloud-sdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5/ConEmu/Scripts:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5/ConEmu:/mnt/c/Python37/Scripts:/mnt/c/Python37:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/iCLS:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/iCLS:/mnt/c/Windows/System32:/mnt/c/Windows:/mnt/c/Windows/System32/wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/IPT:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/IPT:/mnt/c/Program Files/Intel/WiFi/bin:/mnt/c/Program Files/Common Files/Intel/WirelessCommon:/mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/mnt/c/ProgramData/chocolatey/bin:/mnt/c/Program Files/Git/cmd:/mnt/c/Program Files/LLVM/bin:/mnt/c/Windows/System32:/mnt/c/Windows:/mnt/c/Windows/System32/wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Windows/System32/OpenSSH:/mnt/c/Program Files/nodejs:/mnt/c/Users/matt/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/matt/AppData/Local/Programs/Microsoft VS Code/bin:/mnt/c/Users/matt/AppData/Roaming/npm:/snap/bin:/mnt/c/Program Files/Oracle/VirtualBox', 'PS1': '(env3.6) \\[\\e]0;\\u@\\h: \\w\\a\\]${debian_chroot:+($debian_chroot)}\\[\\033[01;32m\\]\\u@\\h\\[\\033[00m\\]:\\[\\033[01;34m\\]\\w\\[\\033[00m\\]\\$ ', 'VAGRANT_HOME': '/home/matt/.vagrant.d/', 'LESSOPEN': '| /usr/bin/lesspipe %s', '_': '/home/matt/repos/cython-blis/env3.6/bin/python'}
    clang -c C:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\blis\_src\config\bulldozer\bli_cntx_init_bulldozer.c -o C:\Users\Siri\AppData\Local\Temp\tmp4yklfekv\bli_cntx_init_bulldozer.o -O2 -funroll-all-loops -std=c99 -D_POSIX_C_SOURCE=200112L -DBLIS_VERSION_STRING="0.5.0-6" -DBLIS_IS_BUILDING_LIBRARY -Iinclude\windows-x86_64 -I.\frame\3\ -I.\frame\ind\ukernels\ -I.\frame\1m\ -I.\frame\1f\ -I.\frame\1\ -I.\frame\include -IC:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\blis\_src\include\windows-x86_64
    ----------------------------------------
    ERROR: Failed building wheel for blis
    Running setup.py clean for blis
  Successfully built preshed
  Failed to build blis
  ERROR: spacy 2.2.4 has requirement blis<0.5.0,>=0.4.0, but you'll have blis 0.2.4 which is incompatible.
  ERROR: spacy 2.2.4 has requirement preshed<3.1.0,>=3.0.2, but you'll have preshed 2.0.1 which is incompatible.
  ERROR: spacy 2.2.4 has requirement thinc==7.4.0, but you'll have thinc 7.0.8 which is incompatible.
  ERROR: chatterbot 1.1.0 has requirement pyyaml<5.2,>=5.1, but you'll have pyyaml 3.12 which is incompatible.
  ERROR: chatterbot 1.1.0 has requirement spacy<2.2,>=2.1, but you'll have spacy 2.2.4 which is incompatible.
  Installing collected packages: setuptools, wheel, Cython, cymem, preshed, murmurhash, numpy, blis, srsly, plac, wasabi, tqdm, thinc
      Running setup.py install for blis: started
      Running setup.py install for blis: finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: 'c:\users\siri\appdata\local\programs\python\python38\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"'; __file__='"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Siri\AppData\Local\Temp\pip-record-x5kkmtt8\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay' --compile --install-headers 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay\Include\blis'
           cwd: C:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\
      Complete output (25 lines):
      BLIS_COMPILER? None
      running install
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-3.8
      creating build\lib.win-amd64-3.8\blis
      copying blis\about.py -> build\lib.win-amd64-3.8\blis
      copying blis\benchmark.py -> build\lib.win-amd64-3.8\blis
      copying blis\__init__.py -> build\lib.win-amd64-3.8\blis
      creating build\lib.win-amd64-3.8\blis\tests
      copying blis\tests\common.py -> build\lib.win-amd64-3.8\blis\tests
      copying blis\tests\test_dotv.py -> build\lib.win-amd64-3.8\blis\tests
      copying blis\tests\test_gemm.py -> build\lib.win-amd64-3.8\blis\tests
      copying blis\tests\__init__.py -> build\lib.win-amd64-3.8\blis\tests
      copying blis\cy.pyx -> build\lib.win-amd64-3.8\blis
      copying blis\py.pyx -> build\lib.win-amd64-3.8\blis
      copying blis\cy.pxd -> build\lib.win-amd64-3.8\blis
      copying blis\__init__.pxd -> build\lib.win-amd64-3.8\blis
      running build_ext
      error: [WinError 2] The system cannot find the file specified
      msvc
      py_compiler msvc
      {'LS_COLORS': 'rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.Z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:', 'HOSTTYPE': 'x86_64', 'LESSCLOSE': '/usr/bin/lesspipe %s %s', 'LANG': 'C.UTF-8', 'OLDPWD': '/home/matt/repos/flame-blis', 'VIRTUAL_ENV': '/home/matt/repos/cython-blis/env3.6', 'USER': 'matt', 'PWD': '/home/matt/repos/cython-blis', 'HOME': '/home/matt', 'NAME': 'LAPTOP-OMKOB3VM', 'XDG_DATA_DIRS': '/usr/local/share:/usr/share:/var/lib/snapd/desktop', 'SHELL': '/bin/bash', 'TERM': 'xterm-256color', 'SHLVL': '1', 'LOGNAME': 'matt', 'PATH': '/home/matt/repos/cython-blis/env3.6/bin:/tmp/google-cloud-sdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5/ConEmu/Scripts:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5:/mnt/c/Users/matt/Documents/cmder/vendor/conemu-maximus5/ConEmu:/mnt/c/Python37/Scripts:/mnt/c/Python37:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/iCLS:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/iCLS:/mnt/c/Windows/System32:/mnt/c/Windows:/mnt/c/Windows/System32/wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/DAL:/mnt/c/Program Files (x86)/Intel/Intel(R) Management Engine Components/IPT:/mnt/c/Program Files/Intel/Intel(R) Management Engine Components/IPT:/mnt/c/Program Files/Intel/WiFi/bin:/mnt/c/Program Files/Common Files/Intel/WirelessCommon:/mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/mnt/c/ProgramData/chocolatey/bin:/mnt/c/Program Files/Git/cmd:/mnt/c/Program Files/LLVM/bin:/mnt/c/Windows/System32:/mnt/c/Windows:/mnt/c/Windows/System32/wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Windows/System32/OpenSSH:/mnt/c/Program Files/nodejs:/mnt/c/Users/matt/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/matt/AppData/Local/Programs/Microsoft VS Code/bin:/mnt/c/Users/matt/AppData/Roaming/npm:/snap/bin:/mnt/c/Program Files/Oracle/VirtualBox', 'PS1': '(env3.6) \\[\\e]0;\\u@\\h: \\w\\a\\]${debian_chroot:+($debian_chroot)}\\[\\033[01;32m\\]\\u@\\h\\[\\033[00m\\]:\\[\\033[01;34m\\]\\w\\[\\033[00m\\]\\$ ', 'VAGRANT_HOME': '/home/matt/.vagrant.d/', 'LESSOPEN': '| /usr/bin/lesspipe %s', '_': '/home/matt/repos/cython-blis/env3.6/bin/python'}
      clang -c C:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\blis\_src\config\bulldozer\bli_cntx_init_bulldozer.c -o C:\Users\Siri\AppData\Local\Temp\tmpzoqx5ou9\bli_cntx_init_bulldozer.o -O2 -funroll-all-loops -std=c99 -D_POSIX_C_SOURCE=200112L -DBLIS_VERSION_STRING="0.5.0-6" -DBLIS_IS_BUILDING_LIBRARY -Iinclude\windows-x86_64 -I.\frame\3\ -I.\frame\ind\ukernels\ -I.\frame\1m\ -I.\frame\1f\ -I.\frame\1\ -I.\frame\include -IC:\Users\Siri\AppData\Local\Temp\pip-install-x9tj7_tg\blis\blis\_src\include\windows-x86_64
      ----------------------------------------
  ERROR: Command errored out with exit status 1: 'c:\users\siri\appdata\local\programs\python\python38\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"'; __file__='"'"'C:\\Users\\Siri\\AppData\\Local\\Temp\\pip-install-x9tj7_tg\\blis\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Siri\AppData\Local\Temp\pip-record-x5kkmtt8\install-record.txt' --single-version-externally-managed --prefix 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay' --compile --install-headers 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay\Include\blis' Check the logs for full command output.
  ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\siri\appdata\local\programs\python\python38\python.exe' 'c:\users\siri\appdata\local\programs\python\python38\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\Siri\AppData\Local\Temp\pip-build-env-q4h7xxqu\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools 'wheel>0.32.0,<0.33.0' Cython 'cymem>=2.0.2,<2.1.0' 'preshed>=2.0.1,<2.1.0' 'murmurhash>=0.28.0,<1.1.0' 'thinc>=7.0.8,<7.1.0' Check the logs for full command output.

**Comment by pavankumar1809** [comment]
i am getting this error  plz any one help me out of this

**Comment by jainshikha** [comment]
getting the same can anyone help on this

**Comment by Jinendra-Jain** [comment]
Getting same type of error

**Comment by novaintegra** [comment]
Getting same type of error

**Comment by danielxjean** [comment]
I resolved it using the previous version of chatterbot: pip install chatterbot ==1.0.4
Look at this thread for more information: https://www.bountysource.com/issues/83146413-pip-install-chatterbot-error

**Comment by gunthercox** [comment]
I believe this installation issue may have been resolved with #2027. Please let me know or open a new ticket if issues persist.
